hydro_lang/src/live_collections/keyed_singleton.rs

```diff
@@ -924,6 +924,68 @@ impl<'a, K: Hash + Eq, V, L: Location<'a>> KeyedSingleton<K, V, Tick<L>, Bounded
             .into_keyed()
     }
 
+    /// Given a keyed stream of lookup requests, where the key is the lookup and the value
+    /// is some additional metadata, emits a keyed stream of lookup results where the key
+    /// is the same as before, but the value is a tuple of the lookup result (as `Option<V>`)
+    /// and the metadata of the request. Unlike `get_many_if_present`, this returns all request
+    /// keys, with `None` for keys that are not found.
+    ///
+    /// This is an "outer join" operation that is both more efficient and more elegant than
```

Comment:
Can probably drop this statement. At the least, it's not more efficient for now since you are doing the anti join under the hood :)